ppr-ui/src/components/unitNotes/UnitNotePanel.vue

```diff
@@ -132,13 +132,24 @@ export default defineComponent({
         case UnitNoteDocTypes.NOTE_CANCELLATION:
           cancelUnitNote(note)
           break
+        case UnitNoteDocTypes.NOTICE_OF_REDEMPTION:
+          setMhrUnitNote(prefillUnitNote(note, option))
+          router.push({ path: '/' + RouteNames.MHR_INFORMATION_NOTE })
```

Comment:
You can push to named route without concatenating. 
Or `goToRoute()` from `useNavigation` etc.

Reply:
I tried `useNavigation`, but it caused strange errors in the tests, which I wasn't able to fix quickly.